modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+[IMPORTANT]
+====
+Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Use the following procedure to create discovery requests, automatically discover local devices, and automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To create a local volume discovery request from the web console:
+
+.. Navigate to *Operators* -> *OperatorHub* -> *Installed Operators*.
+
+.. Click on *Local Storage* from the Operators list.
+
+.. Select the *Local Volume Discovery Result* tab to display the list of results.
+
+.. Click *Create Local Volume Discovery Result*.
+
+.. Complete the required fields in *Form View* or click *YAML view* for full control.
+
+.. When finished, click *Create*. The new result is displayed.
+. To display a continuous list of available devices on a node:
+
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1887008
https://bugzilla.redhat.com/show_bug.cgi?id=1887597
Customer no need to create localvolumediscoberyresult
And for 4.6, we should clarify it here.

Reply:
Thanks, I can't tell from the bugs if the UI will hide this at release, or is a cluster admin supposed to follow these instructions? https://docs.openshift.com/container-platform/4.4/operators/operator_sdk/osdk-generating-csvs.html#osdk-hiding-internal-objects_osdk-generating-csvs

Reply:
I will remove the procedure. If the tab is still shown in the UI, then we should consider whether it should be called out that the admin should hide it from the UI following these instructions: https://docs.openshift.com/container-platform/4.5/operators/operator_sdk/osdk-generating-csvs.html#osdk-hiding-internal-objects_osdk-generating-csvs